Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1136,10 +1136,15 @@ if needed.
 The source for the affinity depends on the origin of a request. If a request is
 sent from outside the cluster to the service, the request's source IP address is
 used for determining the endpoint affinity. If a request is sent from inside
-the cluster, the client's network namespace cookie is used. The latter was introduced
-in the 5.7 Linux kernel to implement the affinity at the socket layer at which
-:ref:`host-services` operate (a source IP is not available there, as the endpoint
-selection happens before a network packet has been built by the kernel).
+the cluster, then the source depends on whether the :ref:`host-services` feature
+is used to load balance ClusterIP services. If yes, then the client's network
+namespace cookie is used as the source. The latter was introduced in the 5.7
+Linux kernel to implement the affinity at the socket layer at which
+:ref:`host-services` operate (a source IP is not available there, as the
+endpoint selection happens before a network packet has been built by the
+kernel). If :ref:`host-services` is not used (i.e. the loadbalancing is done by
+the BPF program ``bpf_lxc`` attached to a pod network interface), then the
```

Comment:
Maybe we can avoid references to BPF program names in a documentation destined to users?
```suggestion
kernel). If :ref:`host-services` is not used (i.e. the loadbalancing is done at
the pod network interface, on a per-packet basis), then the
```